src/components/WelcomeView.jsx

```diff
@@ -82,11 +83,16 @@ class WelcomeView extends React.PureComponent {
           Welcome to the Jade Data Repository
           </div>
           <div>
-            <GoogleLogin // stealing clientId from Terra
+            <GoogleLogin // TODO this component may be unuseable once we require a terra registration
+              // stealing clientId from Terra
               clientId="500025638838-s2v23ar3spugtd5t2v1vgfa2sp7ppg0d.apps.googleusercontent.com"
               buttonText="Sign in with Google"
               onSuccess={onSignInSuccess}
               onFailure={onSignInFailure}
+              isSignedIn
+              cookiePolicy={'single_host_origin'}
```

Comment:
What are the other options here? Thinking we will want to pick something that makes it possible to switch here from terra without re-logging in.

Reply:
I'm guessing I will need to re-write this anyway once SAM has been hooked up.